src/core/util.ts

```diff
@@ -1,12 +1,12 @@
 import { Handle } from './Destroyable';
-import { DNode, RenderResult } from './interfaces';
+import { DNode, RenderResult, VNode, WNode } from './interfaces';
 import { isWNode, isVNode } from './vdom';
 
 const slice = Array.prototype.slice;
 const hasOwnProperty = Object.prototype.hasOwnProperty;
 
 export interface Modifier<T extends DNode> {
-	(dNode: T, breaker: () => void): void;
+	(dNode: T, breaker: () => void): void | VNode | WNode | null | string | boolean;
```

Comment:
should this not just be `void | DNode`?

Reply:
The reason I didn't do that was because it's checking if it's a new node based on whether the return is strictly equal to `undefined`. `DNode` includes undefined so if they returned `undefined` as a new node it wouldn't be picked up.

If that's a problem I could add another callback to pass the new node to instead or something to avoid the ambiguity?

Reply:
Hmmm, it seems like we'd want to support being able to return `undefined` right?